### ISSUE TITLE
[BUG] Cannot set systemProperties for the Gradle plugin

### ISSUE THREAD

**Issue by alwaysbemark** [OpenAPITools/openapi-generator#10181]
##### Description

Hello, I am trying to set the systemProperties field of the Gradle plugin's `openApiGenerate` task and am getting the following error:

```
Caused by: groovy.lang.MissingPropertyException: Could not set unknown property 'systemProperties' for extension 'openApiGenerate' of type org.openapitools.generator.gradle.plugin.extensions.OpenApiGeneratorGenerateExtension.
	at org.gradle.internal.metaobject.AbstractDynamicObject.setMissingProperty(AbstractDynamicObject.java:118)
```

I need this to set generateSupportingFiles to false. It seems that the maven plugin has this as a property but for Gradle it has to be done through a system property (is it better perhaps to ask for that as a feature instead?).

This is my Gradle task definition:

```
openApiGenerate {
    generatorName = "spring"
    inputSpec = "$rootDir/api/api-v1.0.yml".toString()
    outputDir = "$projectDir".toString()
    apiPackage = "com.example"
    modelPackage = "com.example"
    configOptions = [
            interfaceOnly: "true",
            openApiNullable: "false",
    ]
    systemProperties = [
            generateSupportingFiles: "false"
    ]
}
``` 

Thank you!

##### openapi-generator version

5.2.1
(could be a regression of https://github.com/OpenAPITools/openapi-generator/issues/551)

##### Steps to reproduce

Add my task configuration to a blank Gradle 7.1.1 project.

##### Related issues/PRs

https://github.com/OpenAPITools/openapi-generator/issues/551

**Comment by Stiuil06** [comment]
@alwaysbemark 
did you try `globalProperties` or  `additionalProperties` instead?
or maybe you somehow find other solution ?

**Comment by alwaysbemark** [comment]
> @alwaysbemark did you try `globalProperties` or `additionalProperties` instead? or maybe you somehow find other solution ?

Yup, my solution is described here: https://stackoverflow.com/a/68855372/597973
